lib/public/views/Logs/Logs.js

```diff
@@ -82,91 +110,24 @@ export default class Overview extends Observable {
         const result = await response.json();
 
         if (result.data) {
-            const logId = result.data.id;
+            this.createdLog = RemoteData.success(result.data);
             this.title = '';
             this.text = '';
-            this.error = false;
 
-            await this.fetchOneLog(logId);
-            await this.model.router.go(`/?page=entry&id=${logId}`);
+            await this.model.router.go(`/?page=entry&id=${this.createdLog.payload.id}`);
             this.notify();
```

Comment:
Is this notify still needed if we are going to redirect?

Reply:
I had a look, and it did not seem detrimental - removed